modules/AOW_WorkFlow/aow_utils.php

```diff
@@ -235,7 +235,7 @@ function getValidFieldsTypes($module, $field){
 }
 
 
-function getModuleField($module, $fieldname, $aow_field, $view='EditView',$value = '', $alt_type = '', $currency_id = '', $params= array()){
+function getModuleField($module, $fieldname, $aow_field, $view='EditView',$value = '', $alt_type = '', $currency_id = '', $params= array(), $report_csv){
```

Comment:
1. "$report_csv" needs to be "$reports_csv = false" to prevent PHP error "Missing argument 9 for getModuleField(), called in AOR_Report.php on line 753".

2. PSR-2, { needs to be on the next line.

3. Not a requirement but it would be helpful to include a PHPDoc above the function. E.g.
```
/**
 * This function retrieves a module's fields.
 *
 * @param string $module
 * @param string $fieldname
 * @param string $aow_field
 * @param string $view name
 * @param string $value
 * @param string $alt_type
 * @param string $currency_id
 * @param array $params
 * @param bool $reports_csv determines if returned field is for building a csv
 * @return string module fields
 */
```

Reply:
4. Again not a requirement but you could update the [license and entry point](https://suitecrm.com/wiki/index.php/Coding_Standards) at the top of this file.